### PR TITLE
add Will Binns-Smith

### PR DESCRIPTION
**Pull request by wbinnssmith**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -49,7 +49,7 @@
 - [Matthew Mueller](https://github.com/matthewmueller/ama) - Creator of Cheerio, Duo, and a few other node modules.
 - [DeokHong Kim](https://github.com/insanehong/ama) - Front-end Engineer at NAVER, founder of [hackrslab](https://github.com/hackrslab).
 - [Hugo Giraudel](https://github.com/hugogiraudel/ama) - CSS goblin, Sass hacker, margin psycho.
-
```
Comment (arthurvr) [inline readme.md:52]: Don't remove this line :) I fixed it for you while merging :wink:

Comment (wbinnssmith) [inline readme.md:52]: Appreciated :)

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Comment by wbinnssmith** [comment]
Absolutely. Coming up soon.

**Pull request closed without merging**